src/tfr_control/arduino/encoder/encoder.ino

```diff
@@ -2,93 +2,120 @@
 #include <Wire.h>
 #include <Adafruit_ADS1015.h>
 #include <ros.h>
-#include <tfr_msgs/EncoderReading.h>
-#include <tfr_msgs/PotentiometerReading.h>
+#include <tfr_msgs/ArduinoReading.h>
 #include <quadrature.h>
 
 ros::NodeHandle nh;
 
 //encoder level constants
-const double GEARBOX_CPR = 4096; //pulse per revolution
-//TODO note this rev/m is acually 0.87, but I had to step it up to make testing work
+const double CPR = 4096; //pulse per revolution
+//TODO note this rev/m is acually 0.876, but I had to step it up to make testing work
 const double GEARBOX_RPM = 60; //revolutions per meter
+//TODO note this rev/m is acually 0.208, but I had to step it up to make testing work
+const double TURNTABLE_RPM = 60;
 
 //pin constants
 const int GEARBOX_LEFT_A = 2;
 const int GEARBOX_LEFT_B = 3;
 
-struct potentiometer
+
+
+
+struct Potentiometer
 {
-    const static float ADC_RANGE=0.0001875;
-    float extension_range; //the max range the potentiometer can be extended
-    float max_voltage; //the max voltage for the potentiometer
-    float getExtension(int16_t val)
-    {
-        return val* ADC_RANGE/max_voltage; 
-    }
+   const static float ADC_RANGE=0.0001875;
+   float max_voltage; //the max voltage for the potentiometer
```

Comment:
Does the mixing of floats and doubles have a reason? (could everything be a double? I know you mentioned some library calls need it) It bugs me stylistically a bit, but if there's reasons for it then it's fine.